lib/ClangImporter/ClangAdapter.cpp

```diff
@@ -712,7 +712,7 @@ OptionalTypeKind importer::getParamOptionality(version::Version swiftVersion,
     return OTK_None;
 
   // Check for the 'static' annotation on C arrays.
-  if (!swiftVersion.isVersion3())
+  if (swiftVersion != version::Version({3}))
```

Comment:
I would prefer to leave this exactly as is, unperturbed and add the new utility function for future use.  What do you all think?  This means, this is reverted to what it was.